tests/test-class-amp-theme-support.php

```diff
@@ -212,6 +212,49 @@ public function test_prepare_response() {
 		$this->assertInstanceOf( 'DOMAttr', $removed_nodes['onclick'] );
 	}
 
+	/**
+	 * Test validate_non_amp_theme.
+	 *
+	 * @global WP_Widget_Factory $wp_widget_factory
+	 * @global WP_Scripts $wp_scripts
+	 * @covers AMP_Theme_Support::prepare_response()
+	 */
+	public function test_validate_non_amp_theme() {
+		global $wp_widget_factory, $wp_scripts;
```

Comment:
`$wp_widget_factory` doesn't seem to be used in this method and the `$wp_scripts` scripts are already reset in the `tearDown()` method.